suite-common/message-system/config/config.v1.json

```diff
@@ -1139,6 +1139,76 @@
                 }
             }
         },
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": "*",
+                        "mobile": "!",
+                        "web": "*"
+                    },
+                    "devices": [
+                        {
+                            "model": "1",
+                            "firmware": "1.5.1>1.7.0",
+                            "bootloader": "*",
+                            "variant": "*",
+                            "firmwareRevision": "*",
+                            "vendor": "*"
+                        },
+                        {
+                            "model": "T1B1",
+                            "firmware": "1.5.1>1.7.0",
```

Comment:
cc @tomasklim @Lemonexe - just checking whether I can use the conditions like this

Reply:
nope it is not valid

Reply:
greater than or equal to 1.5.1 and less than 1.7.0
`"firmware": ">=1.5.1 <1.7.0"`

Reply:
Solved in e38213bf7dc8faab8371b673e75e084125f6d301